content/docs/pipelines/pipelines-deploy-alternatives.md

```diff
@@ -0,0 +1,139 @@
++++
+title = "Pipelines Deploy Alternatives"
+description = "Alternative approaches to deploy lightweight Kubeflow Pipelines to a cluster"
+weight = 15
++++
+
+Despite [deploying Kubeflow](/docs/started/getting-started/#installing-kubeflow) as a
+whole with many components including pipelines, you also have a choice to deploy
+Kubeflow Pipelines' lite deployment. Pipelines has a kustomize manifests for
+this. You can follow the instructions below to deploy it in an existing cluster.
+
+Knowledge about [Kubernetes](https://kubernetes.io/docs/home/), [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) and [kustomize](https://kustomize.io/) will help you understand this
+document better and be able to customize based on your needs.
+
+
+## TL;DR
+
+To deploy Kubeflow Pipelines in an existing Kubernetes cluster, do the following:
+
+1. Deploy latest version of Kubeflow Pipelines
```

Comment:
For readability, add a colon ":" at the end of this line and every similar line below.